docs/pages/includes/determinetls.mdx

```diff
@@ -0,0 +1,26 @@
+<Details title="How to tell if TLS Routing is Enabled" opened={false}>
```

Comment:
I think we can remove the `Details` wrapper, since it could be helpful to confirm whether TLS Routing is enabled or not in all cases, and doesn't add much time.